_posts/fr/newsletters/2024-08-16-newsletter.md

```diff
@@ -150,9 +150,9 @@ apportés aux principaux logiciels d'infrastructure Bitcoin.
   Des retours sur les méthodes cryptographiques à utiliser sont demandés, avec Teinturier prévoyant de
   publier [BLIP42][blips #42] avec une spécification pour les méthodes sélectionnées.
 
-- **Passage de Bitcoin Core au système de construction CMake :** Cory Fields a [posté][fields cmake]
+- **Passage de Bitcoin Core au système de compilation CMake :** Cory Fields a [posté][fields cmake]
```

Comment:
Youll need to update the link from Newsletter #319 to this news item to fix the build